core/src/zeit/connector/postgresql.py

```diff
@@ -762,6 +776,20 @@ def from_webdav(self, props):
             unsorted[ns.replace(self.NS, '', 1)][k] = v
         self.unsorted = unsorted
 
+    def _set_checksum(self):
+        """but now we have to get the body every time, we use to_webdav, sigh"""
+        if self.is_collection:
+            return None
+
+        alg = hashlib.md5(usedforsecurity=False)
+        meta = json.dumps(sorted(self.unsorted.items()), ensure_ascii=False)
+        if self.binary_body:
+            alg.update(meta.encode('utf-8'))
+            return alg.hexdigest()
+        text = f'{meta}{self.body}'
+        alg.update(text.encode('utf-8'))
+        return alg.hexdigest()
```

Comment:
`alg.update(meta.encode('utf-8'))` können wir hoch schieben. Die Operation brauchen wir ja auf jeden Fall :). Den body können wir dann einfach der Checksumme hinzufügen. Heißt wir sparen auch wieder die Operation den str zu kombinieren 🥳 

```suggestion
        alg.update(meta.encode('utf-8'))
        if self.binary_body:
            return alg.hexdigest()

        alg.update(self.body.encode('utf-8'))
        return alg.hexdigest()
```